f2/src/LandingPage.js

```diff
@@ -15,6 +15,17 @@ import { TrackPageViews } from './TrackPageViews'
 import { Stack, Icon, Alert, AlertIcon, Box } from '@chakra-ui/core'
 import { useStateValue } from './utils/state'
 import { LandingBox } from './components/container'
+import { i18n } from '@lingui/core'
+
+if (
+  window.location.hostname ===
+  'https://signalez-un-crime-informatique.cds-snc.ca/'
+) {
+  i18n.activate('fr')
+} else if (window.location.hostname === 'report-a-scam.cds-snc.ca') {
+  i18n.activate('en')
+} else {
+}
```

Comment:
We don't need the `else {}` here

Reply:
Also, on Chrome in the console I'm just getting `signalez-un-crime-informatique.cds-snc.ca` for the hostname:
![image](https://user-images.githubusercontent.com/8228248/74676357-df787300-5183-11ea-85b5-831d46b839a4.png)
It might be better to do something like
```
if (window.location.hostname .indexOf('signalez')> -1) ....
else if (window.location.hostname .indexOf('report')> -1) ...